_posts/2022-07-25-changelog-175.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #175"
+date: 2022-07-25 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8767] You can now paste JSON objects into Rust files and convert them to a set of Rust structs (by [@Kobzol])
```

Comment:
Maybe it's worth to add all PRs related to this feature?

Reply:
Also, it would be great to mention (and maybe show) that feature can add `serde` attributes